compiler/src/printCommon.ml

```diff
@@ -151,7 +151,7 @@ let string_of_op2 = function
   | Owi2(sg, ws, o) -> string_of_wiop2 sg ws o
 
 (* -------------------------------------------------------------------- *)
-let pp_opn pd asmOp fmt o = pp_string fmt (Sopn.string_of_sopn pd asmOp o)
+let pp_opn pd asmOp fmt o = pp_string fmt (Sopn.string_of_sopn pd pd asmOp o)
```

Comment:
Again strange, the original one corresponds to msf_size ?

Reply:
That’s true. This is addressed in the latest commit.